docs/source/UserGuide/index.rst

```diff
@@ -0,0 +1,9 @@
+USER GUIDE
```

Comment:
Only thing I'd say here is that I wonder if this page should be removed and just replaced with VUDRM-Admin.md itself. This page is pretty barebones and it seems unneccessary when its linking to a single page.

But if you plan to add more pages to the user guide, then I'm happy for this to stay this way

Reply:
Removed.